README.md

```diff
@@ -1 +1,5 @@
-# Review-past-project
\ No newline at end of file
+# FCC Test Suite Template
```

Comment:
The name must be the same as it is in the project.

Reply:
- It would be better to suggest bringing the changes, ordering is not a good of communicating.